components/typography/demo/typography.html

```diff
@@ -6,6 +6,7 @@
 		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
 		<script type="module">
 			import '../../demo/demo-page.js';
+			import '../typography.js';
```

Comment:
Did you run into a problem that required this import.  It should already be imported by demo-page [here](https://github.com/BrightspaceUI/core/blob/main/components/demo/demo-page.js#L4).

Reply:
Nope added when I was trying to figure something out and forgot about it. Will remove